control/consistency_checker.py

```diff
@@ -0,0 +1,80 @@
+import math
+
+import yaml
+
+
+def read_yaml(file_path):
+    """Read YAML file and return the data."""
+    with open(file_path, "r") as file:
+        return yaml.safe_load(file)
+
+
+# paths to the YAML files
+mpc_param_file_path = "/home/zheshen/pilot-auto.x2/src/autoware/launcher/autoware_launch/config/control/trajectory_follower/lateral/mpc.param.yaml"
+pid_param_file_path = "/home/zheshen/pilot-auto.x2/src/autoware/launcher/autoware_launch/config/control/trajectory_follower/longitudinal/pid.param.yaml"
+simulator_model_param_file_path = "/home/zheshen/pilot-auto.x2/src/description/vehicle/j6_gen1_description/j6_gen1_description/config/simulator_model.param.yaml"
```

Comment:
How about passing these paths as arguments ? For example using `argparse` (https://docs.python.org/3/library/argparse.html).

Reply:
fixed in 40f63540ef837fdeb66d3c6a0463de037bd0b3f9
##### Default:
![image](https://github.com/autowarefoundation/autoware_tools/assets/80969277/22cc1c4c-085f-4ec2-8cac-0ca7529ac048)
##### Override:
![image](https://github.com/autowarefoundation/autoware_tools/assets/80969277/ba1391fc-0fab-4a47-9830-c758a2081e4e)